### DISCUSSION TITLE
Certification of student development package

### DISCUSSION THREAD

**Discussion by himicoswilson** [community/community#84090] [GitHub Education]
### Hi everyone,

I had many certifications in April last year, and was later asked to re-verify. Now I want to re-verify, but the website does not allow me to re-verify, so I can only send an email for help. I hope you can answer my questions. I can provide my admission notice and the original student ID card, as well as some daily selfies of school life. thank you.

**Comment by Pardeep-Kumar0** [comment]
### 1. > [`__`](url)
